.stylish-haskell.yaml

```diff
@@ -198,7 +198,7 @@ steps:
 # Note: we tend to write code which fits into 80 characters, in some cases
 # 100 is acceptable. For imports we always permit 100 characters because it
 # decreases verbosity of diffs and makes merging easier.
-columns: 100
+columns: 80
```

Comment:
I really don't think we should change that one. 80 characters is super narrow and the change is actually un-ncessary.

Reply:
Hmm, 80 chars is the codebase-wide recommended style - it's only here that it was inconsistent. What does everyone else think?